src/components/homePage/SchedulePreview.vue

```diff
@@ -13,4 +51,94 @@ export default {
 </script>
 
 <style scoped>
+.schedule-link {
+	color: var(--main-color);
+	text-decoration: underline;
+}
+.header {
+	display: flex;
+	align-items: baseline;
+	column-gap: 40px;
+}
+.content{
+	display: grid;
+	grid-template-columns: 40% 55%;
+	grid-auto-rows: 50% 50%;
+	grid-gap: 0% 4%;
+	grid-template-areas: 
+		"pink blue"
+		"pink green";
+}
+.box{
+	border-radius: 10px;
+	display: flex;
+}
+.pink{
+	background-color: #ED7C77;
+	width: 100%;
+	grid-area: pink;
+	flex-direction: column;
+}
+.blue{
+	background-color: #0094B5;
+	width: 100%;
+	grid-area: blue;
+	flex-direction: row;
+	margin-bottom: 5%;
+}
+.green{
+	background-color: #44AF69;
+	width: 100%;
+	grid-area: green;
+	flex-direction: row-reverse;
+}
+h3{
+	font-weight: bold;
+	padding: 25px 25px 5px 25px;
```

Comment:
Two things to note here:
1. We use rems instead of px - it just keeps everything standardized (and browsers have nice features associated with them)
2. Instead of specifying that all directions are 25px except one, do it the verbose way. It takes one extra line, but it is a lot more readable.
`padding: 25px;
padding-right: 5px;`